plato/draw/pythreejs/Scene.py

```diff
@@ -142,30 +141,33 @@ def remove_primitive(self, primitive, strict=True):
         self._backend_objects['scene'].remove(primitive.threejs_primitive)
         super(Scene, self).remove_primitive(primitive, strict)
 
+    def _remove_lights(self):
+        for _ in range(len(self._backend_objects['directional_lights'])):
+            light = self._backend_objects['directional_lights'].pop()
+            self._backend_objects['camera'].remove(light)
+
+    def disable(self, name, strict=True):
+        super(Scene, self).disable(name, strict)
```

Comment:
@bdice Disabling `directional_light` will fail now if `strict` is `True`, right? Maybe just do the 'directional_light' check below first, and set `strict` to `False` inside that if-block?

Reply:
Oh, I had forgotten how the base `Scene.disable` worked. This actually looks fine, of course.